woudc_data_registry/peer/file_indices_extractor.py

```diff
@@ -0,0 +1,92 @@
+import os, csv, logging, psycopg2
+from urllib.request import urlopen
+
+LOGGER = logging.getLogger(__name__)
+
+Process=dict(
+  # WOUDC platform station lookup
+  query="SELECT gaw_id,station_id,station_name_id,station_type from stations;",  
+  station_alias='/woudc_data_registry/peer/Platform_name_variations.txt',
+)
+
+def load_station_metadata():
+    """Load WOUDC station metadata lookup list."""
+    # Query PostreSQL database station table for metadata    
+    connection = psycopg2.connect(user=os.getenv('WDR_DB_USERNAME'),
+                                      password=os.getenv('WDR_DB_PASSWORD'),
+                                      host=os.getenv('WDR_DB_HOST'),
+                                      port=os.getenv('WDR_DB_PORT'),
+                                      database=os.getenv('WDR_DB_NAME'))
+    cursor = connection.cursor()
+    cursor.execute(Process['query'])
+    records = cursor.fetchall()
+
+    cursor.close()
+    connection.close()
+
+    stations = {}
+    for station_row in records:
+        station_nameid = station_row[2]
+        station_name_start = station_nameid.index(':') + 1   
+        station_name = station_nameid[station_name_start:len(station_nameid)]
+        stations[station_name.lower()] = (
+                station_row[3], # station_type
+                station_row[1], # station_id
+                station_row[0]) # gaw_id
+
+    return stations
+
+
+def get_station_metadata(name, stations, variations):
+    """Look up station metadata given a station name."""
+    key = name 
+    if key is not None:
+        key = name.lower()
+    if key in stations:
+        return stations[key]
+    elif variations is not None:
+        msg = 'Station metadata not found for the station name: %s' % name
```

Comment:
change to `msg = 'Station metadata not found for the station name: {}'.format(name)`.  Apply to all string interpolation in this PR.